lib/spoom/deadcode/plugins/base.rb

```diff
@@ -35,6 +35,24 @@ def ignore_class_names(*names)
             save_names_and_patterns(names, :@ignored_class_names, :@ignored_class_patterns)
           end
 
+          # Mark classes directly subclassing a class matching `names` as ignored.
+          #
+          # Names can be either strings or regexps:
+          #
+          # ~~~rb
+          # class MyPlugin < Spoom::Deadcode::Plugins::Base
+          #   ignore_subclasses_of(
+          #     "Foo",
+          #     "Bar",
+          #     /Baz.*/,
+          #   )
+          # end
+          # ~~~
+          sig { params(names: T.any(String, Regexp)).void }
+          def ignore_subclasses_of(*names)
```

Comment:
Nit -- do you think `ignore_direct_subclasses_of` would be clearer and match the description of the method?

Reply:
@st0012 raised the same concern about the name and we went with `ignore_classes_inheriting_from`.

But I agree with you Emily, the "direct" seems like an important information here.

On the other hand, maybe I can remove this restriction soon by computing the full type hierarchy 🤔

Reply:
Maybe I should also change the name of `ignnore_class_names(...)` to `ignore_classes_named(...)` or something similar?

Reply:
https://github.com/Shopify/spoom/pull/442